modules/sdk/project-templates/project-templates-npm-angular-portlet/build.gradle

```diff
@@ -4,4 +4,11 @@ targetCompatibility = "1.8"
 dependencies {
 	compileOnly group: "com.liferay", name: "com.liferay.project.templates", version: "4.1.7"
 	compileOnly group: "org.apache.maven.archetype", name: "archetype-common", version: "2.4"
+}
+
+generateSampleProject {
+	args "--liferayVersion", "7.1", "--name", "NpmAngularPortlet"
```

Comment:
@john-co project templates is case sensitive. so when you change the name to NpmAngularPortlet, the project will be so. aka, `project-templates-npm-angular-portlet/builds/NpmAngularPortlet` this is where the project will be

Reply:
so lets just do... `--name npm-angular-portlet` and follow the standard naming convention we use throughout liferay-portal